From fe7014d0db6008e8ce2866ee0331ad6605d8ae8a Mon Sep 17 00:00:00 2001
From: "Pan, Kris" <kris.pan@intel.com>
Date: Tue, 25 Aug 2020 16:23:05 +0800
Subject: [PATCH 1206/1214] Revert "FROMGIT: kallsyms: unexport
 kallsyms_lookup_name() and kallsyms_on_each_symbol()"

This reverts commit 8e37367a324263c8c1b31ab14030d6be27b0cc7e.

ERROR: "kallsyms_lookup_name" [drivers/platform/x86/sepdk/sep/sep5.ko] undefined!
scripts/Makefile.modpost:108: recipe for target '__modpost' failed
make[1]: *** [__modpost] Error 1
Makefile:1344: recipe for target 'modules' failed
make: *** [modules] Error 2

Signed-off-by: Pan, Kris <kris.pan@intel.com>
---
 kernel/kallsyms.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/kernel/kallsyms.c b/kernel/kallsyms.c
index b8ec1db..eea45e7 100644
--- a/kernel/kallsyms.c
+++ b/kernel/kallsyms.c
@@ -173,6 +173,7 @@ unsigned long kallsyms_lookup_name(const char *name)
 	}
 	return module_kallsyms_lookup_name(name);
 }
+EXPORT_SYMBOL_GPL(kallsyms_lookup_name);
 
 int kallsyms_on_each_symbol(int (*fn)(void *, const char *, struct module *,
 				      unsigned long),
@@ -191,6 +192,7 @@ int kallsyms_on_each_symbol(int (*fn)(void *, const char *, struct module *,
 	}
 	return module_kallsyms_on_each_symbol(fn, data);
 }
+EXPORT_SYMBOL_GPL(kallsyms_on_each_symbol);
 
 static unsigned long get_symbol_pos(unsigned long addr,
 				    unsigned long *symbolsize,
-- 
2.7.4

